packages/account/src/Configs/get-status-badge-config.tsx

```diff
@@ -3,7 +3,11 @@ import { Link } from 'react-router-dom';
 import { Text } from '@deriv/components';
 import { Localize } from '@deriv/translations';
 
-const getStatusBadgeConfig = (account_status, openFailedVerificationModal, selected_account_type) => {
+const getStatusBadgeConfig = (
+    account_status: string,
+    openFailedVerificationModal: (selected_account_type: string) => void,
+    selected_account_type: string
```

Comment:
Same for these as the possible acceptable values

Reply:
yes, I thought about it, but I'm not aware of type used here and not sure if we can cover all statuses, because function can handle only 3 of them. so I left here string by purpose

Reply:
I would suggest adding the possible accepted values to make it type safe, string can be anything

Reply:
yes, and for this case we have default return

Reply:
same fields for functions are also described in store types, and it is just string everywhere.
if we want to specify them (I believe this could be better), then we need to specify them across the app, and not only here